src/firefly/plans/util.py

```diff
@@ -15,11 +15,14 @@ def time_converter(total_seconds):
     ==========
     tuple: hours, minutes, seconds
     """
+    if np.isnan(total_seconds):
+        return float("nan"), float("nan"), float("nan")
+    # Use np.floor to round to the smaller integer
+    hours = np.floor(total_seconds / 3600).astype(int)
```

Comment:
Haha, good catch! I'm amazed we both looked at this for as long as we did and didn't notice this until now.